math/lcm.go

```diff
@@ -0,0 +1,49 @@
+// Formula from https://en.wikipedia.org/wiki/Least_common_multiple
+// Find the least common multiple of two integers
+// by using the greatest common divisor.
+
+package main
+
+import (
+	"fmt"
+)
+
+// abs: absolute value for an integer
+func abs(a int) int {
+	if a < 0 {
+		return -a
+	}
+	return a
+}
+
+// gcd: greatest common divisor -
+// the largest positive integer that divides both a and b
+func gcd(a, b int) int {
+	if b == 0 {
+		return a
+	}
+	return gcd(b, a%b)
+}
+
+// lcm: least common multiple -
+// the smallest positive integer that is divisible by both a and b
+func lcm(a, b int) int {
```

Comment:
Capitalise the main algorithms function that you created this PR for.

```suggestion
func Lcm(a, b int) int {
```